packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -241,7 +240,7 @@ export default class RelationControl extends React.Component {
     const isMultiple = this.isMultiple();
     const isClearable = !field.get('required', true) || isMultiple;
 
-    const hits = queryHits.get(forID, []);
+    const hits = queryHits[forID] || [];
```

Comment:
What do you think about moving this to https://github.com/netlify/netlify-cms/blob/d057e023640e2076ab8be1f3ce2364a992cdce3e/packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js#L314

Reply:
Not sure I understand the proposal

Reply:
See https://github.com/netlify/netlify-cms/pull/5157/commits/3a68c43697a683911a9a590e5c760f235f5f805f

Reply:
Hm... according to EditorControl propTypes queryHits can be either array or object

Reply:
So in case of array ```queryHits[this.uniqueFieldId]``` won't work

Reply:
Same as `RelationControl` https://github.com/netlify/netlify-cms/blob/3211f94f4ae55cc008bc28325a5387722657cd2c/packages/netlify-cms-widget-relation/src/RelationControl.js#L95

I'll verify it can only be an object

Reply:
Yep, I think with https://github.com/netlify/netlify-cms/pull/5157/commits/96e378134ae232d56ed77a13e7f1748dd7e244ca the prop types now match https://github.com/netlify/netlify-cms/pull/5157/files#diff-f141f1b29eb66150ed0215315c55f8e2d91dae5fb1f947c416475c5d2e4a2028R21